.github/workflows/ci.yml

```diff
@@ -80,3 +82,19 @@ jobs:
         NODE_ENV: disable-logging
       run: |
         npx jest -c jest/integration.config.js "${{ matrix['test-suite'] }}" --bail  --forceExit --detectOpenHandles --runInBand
+  subscribe:
+    needs: ci
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-node@v1
+      with:
+        node-version: '15.3'
+    - name: start latest test node
+      run: |
+        docker run --network=host -d cennznet/cennznet:1.2.0 --dev --tmp --ws-external --rpc-external --no-mdns
+    - name: run subscriber script at rpc-provider (2.7.1)
```

Comment:
since we already have a subscribe e2e test, do we still need to run the example here / what test coverage does it add?

Reply:
subscribe e2e test cannot use fixed/pinned dependency, it will get the dependencies from node_modules..

Reply:
Ok I see now, please leave some comments in the test so we know why it's there